src/test/java/io/strimzi/kafka/bridge/http/ConsumerTest.java

```diff
@@ -1636,6 +1638,6 @@ void setUp() {
 
     private String generateRandomConsumerName() {
         int salt = new Random().nextInt(Integer.MAX_VALUE);
-        return "my-kafka-consumer" + salt;
+        return "my-kafka-consumer" + salt + salt;
```

Comment:
I had a discussion with @see-quick about this. Adding salt two times is useless.